meetings/README.md

```diff
@@ -2,17 +2,17 @@
 
 ## #ansible-meeting
 
-Ansible IRC meetings are held on [irc.libera.chat](https://libera.chat/).
-
-If you receive an error when you connect, see [libera.chat’s Nickname Registration guide](https://libera.chat/guides/registration) for instructions.
+Ansible meetings are held in [Matrix](https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-matrix) and/or bridged [IRC](https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-irc) channels.
```

Comment:
Isn't it (still) the other way around right now, since zodbot is on IRC and not on Matrix?

Reply:
fair point, changed, thanks!